WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Summary/SimplePaymentsNoteViewModel.swift

```diff
@@ -1,63 +1,10 @@
 import Foundation
 import Combine
 
-final class SimplePaymentsNoteViewModel: EditCustomerNoteViewModelProtocol {
-
-    /// Store for the edited content
-    ///
-    @Published var newNote: String
-
-    /// Defines the navigation right button state
-    ///
-    @Published private(set) var navigationTrailingItem: EditCustomerNoteNavigationItem = .done(enabled: false)
-
-    /// Not used.
-    ///
-    @Published var presentNotice: EditCustomerNoteNotice? = nil
-
-    /// Not used.
-    ///
-    var presentNoticePublisher: Published<EditCustomerNoteNotice?>.Publisher {
-        $presentNotice
-    }
-
-    /// Commit the original note.
-    ///
-    func updateNote(onFinish: @escaping (Bool) -> Void) {
-        originalNote = newNote
-        onFinish(true)
-
-        analytics.track(event: WooAnalyticsEvent.SimplePayments.simplePaymentsFlowNoteAdded())
-    }
-
-    /// Revert to original content.
-    ///
-    func userDidCancelFlow() {
-        newNote = originalNote
-    }
-
-    /// Stores the original note content.
-    ///
-    @Published private var originalNote: String
-
-    /// Analytics engine.
-    ///
-    private let analytics: Analytics
-
+class SimplePaymentsNoteViewModel: OrderCreationNotesViewModel {
```

Comment:
I would suggest making the simple payments changes in a different PR! Even if they are small it helps a lot to the reviewers to focus their testing efforts in just one area!

Reply:
Agreed! I've reverted all changes related to SimplePayments refactoring and I'm now leaving the Order Creation Customer note view model as a dedicated one with no code reuse, this way we can adjust everything in a different PR. Thanks!